docs/getting-started-guide/styles/foundation.css

```diff
@@ -0,0 +1,1927 @@
+  font-size: 12px;
+  font-weight: bold;
+  line-height: 20px;
+  font-family: Arial, sans-serif;
+  font-style: normal;
+  position: relative;
+  top: -2px;
+  letter-spacing: -1px
+}
+
+.conum * {
+  color: white !important
+}
+
+.conum + b {
+  display: none
+}
+
+.conum:after {
+  content: attr(data-value)
+}
+
+.conum:not([data-value]):empty {
+  display: none
+}
+
+#content {
+  margin: 20px 20px 0 20px;
+  padding: 10px 10px 60px 10px;
+}
+
+#header {
+  margin: 0 20px;
+}
+
+body:before {
+  content: "";
+  background: no-repeat 0/250px url(../images/RHD-logo.svg) #030303;
+  height: 70px;
+  width: 100%;
+  display: inline-block;
+  padding: 8px 35px;
+}
+
+#footer-text {
+  font-size: smaller;
+  float: right;
+}
+
+.container {
+    margin-right: auto;
+    margin-left: auto;
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+.row {
+    margin-left: -20px;
+    margin-right: -20px;
+}
+
+.col-xs-2 {
+    width: 16.66666667%;
+}
+
+.col-xs-10 {
+    width: 83.33333333%;
+}
+
+.col-xs-1, .col-xs-2, .col-xs-3, .col-xs-4, .col-xs-5, .col-xs-6, .col-xs-7, .col-xs-8, .col-xs-9, .col-xs-10, .col-xs-11, .col-xs-12 {
+    float: left;
+}
+.col-xs-1, .col-sm-1, .col-md-1, .col-lg-1, .col-xs-2, .col-sm-2, .col-md-2, .col-lg-2, .col-xs-3, .col-sm-3, .col-md-3, .col-lg-3, .col-xs-4, .col-sm-4, .col-md-4, .col-lg-4, .col-xs-5, .col-sm-5, .col-md-5, .col-lg-5, .col-xs-6, .col-sm-6, .col-md-6, .col-lg-6, .col-xs-7, .col-sm-7, .col-md-7, .col-lg-7, .col-xs-8, .col-sm-8, .col-md-8, .col-lg-8, .col-xs-9, .col-sm-9, .col-md-9, .col-lg-9, .col-xs-10, .col-sm-10, .col-md-10, .col-lg-10, .col-xs-11, .col-sm-11, .col-md-11, .col-lg-11, .col-xs-12, .col-sm-12, .col-md-12, .col-lg-12 {
+    position: relative;
+    min-height: 1px;
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+footer {
+  background-color: #010101;
+  height: 60px;
+  padding: 15px 0;
+  width: 100%;
+  bottom: 0;
+  left: 0;
+  position: absolute; }
+  footer   .conditions {
+    font-size: 12px; }
+  footer   .conditions   a:nth-child(2) {
+    margin-left: 70px; }
+  footer   .logo   a {
+    background: url(../images/redhat.svg) no-repeat;
+    height: 34px;
+    width: 94px;
+    margin: 0;
+    top: auto;
+    background-size: 94px 30px;
+    display: inline-block; }
+  footer   .summit {
+    float: right; }
+  footer   a {
+    color: #858585;
+    top: 7px;
+    position: relative;
+    margin-left: 15px; }
+    footer   a.legal {
+      border-right: 1px solid #858585;
+      padding-right: 15px; }
+
+@media (max-width: 1024px) {
+  footer   .conditions {
+    font-size: 12px; }
+    .footer-openshiftio   .conditions   a:first-child {
+      margin-left: 35px; }
+  .footer-openshiftio   .social {
+    display: none; } }
+
+@media (max-width: 768px) {
+  footer   .conditions {
+    display: none; }
+  .footer-openshiftio   .summit {
+    position: absolute;
+    right: 20px;
+    bottom: 0; }
+  .footer-openshiftio   a.legal:nth-child(2n) {
+    margin-left: 70px; }
```

Comment:
@rkratky this is the css file I usually get when building the GS doc. Please have a look. I suspect it's fine to just leave it in instead of deleting the folder each time I commit.